spec/controllers/admin/users_controller_spec.rb

```diff
@@ -19,4 +19,24 @@
     end
   end
 
+  describe '#lock_account' do
+    it 'it locks the given account' do
+      other_user = FactoryGirl.create :user
+      expect(other_user).to receive(:lock_access!)
+      allow(User).to receive(:find).and_return(other_user)
```

Comment:
@jhass I didn't get the use of:
```post :lock_account, id: user.id``` initially so I did:
https://github.com/diaspora/diaspora/pull/5643/files#diff-3212b7cc0ec103914d1d59733b84f107R25
I got the whole concept now. I will update the pr.